### ISSUE TITLE
evictor-pvc-pod may not work normally

### ISSUE THREAD

**Issue by hangscer8** [hwameistor/hwameistor#1103]
Output of `lsblk`:
```
[root@localhost ~]# lsblk
NAME                                                                 MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sda                                                                    8:0    0   160G  0 disk
├─sda1                                                                 8:1    0     1G  0 part /boot
└─sda2                                                                 8:2    0   159G  0 part
  ├─centos-root                                                      253:0    0    50G  0 lvm  /
  ├─centos-swap                                                      253:1    0   7.9G  0 lvm
  └─centos-home                                                      253:2    0 101.1G  0 lvm  /home
sdb                                                                    8:16   0   106G  0 disk
├─LocalStorage_PoolHDD-pvc--dae4f576--c03a--479c--ad9c--c107e83b3cc6 253:4    0     2G  0 lvm  /var/lib/kubelet/pods/c1aac5be-e25b-43f6-92cc-5da962a037cd/volumes/kubernetes.io~csi/pvc-dae4f576-c03a-479c-ad9c-c107e83b3cc6/mount
├─LocalStorage_PoolHDD-pvc--5835a84b--20c2--4cac--b677--90836732577d 253:5    0    10G  0 lvm  /var/lib/kubelet/pods/37422833-256b-4a5e-b58d-2179b9d1a629/volumes/kubernetes.io~csi/pvc-5835a84b-20c2-4cac-b677-90836732577d/mount
├─LocalStorage_PoolHDD-pvc--fa14345c--0704--4646--9cb4--a91ee8ee54c9 253:6    0    35G  0 lvm  /var/lib/kubelet/pods/365247a1-b121-4139-8f7a-e96c9ac70914/volumes/kubernetes.io~csi/pvc-fa14345c-0704-4646-9cb4-a91ee8ee54c9/mount
├─LocalStorage_PoolHDD-pvc--d49464f5--00a3--4638--9726--4c059baa12e2 253:7    0    25G  0 lvm  /var/lib/kubelet/pods/2ab03046-e371-445d-90fd-9a6019159273/volumes/kubernetes.io~csi/pvc-d49464f5-00a3-4638-9726-4c059baa12e2/mount
├─LocalStorage_PoolHDD-pvc--8627b40b--44a9--4e69--85a6--bec5c1d9b005 253:8    0    10G  0 lvm  /var/lib/kubelet/pods/8048c952-4796-47f5-9e01-c393c101eae7/volumes/kubernetes.io~csi/pvc-8627b40b-44a9-4e69-85a6-bec5c1d9b005/mount
├─LocalStorage_PoolHDD-pvc--64ed9646--bc61--4d2c--91ff--a6108725f9a2 253:9    0     2G  0 lvm  /var/lib/kubelet/pods/97fa07e0-90f8-4f4b-ba07-ce738f9a8374/volumes/kubernetes.io~csi/pvc-64ed9646-bc61-4d2c-91ff-a6108725f9a2/mount
├─LocalStorage_PoolHDD-pvc--995ef500--6b84--4d65--81f9--425eb9795b7a 253:10   0     8G  0 lvm  /var/lib/kubelet/pods/e949ba40-a5a0-4230-ac38-d5f0ea8301c3/volumes/kubernetes.io~csi/pvc-995ef500-6b84-4d65-81f9-425eb9795b7a/mount
├─LocalStorage_PoolHDD-pvc--b7bc6400--4c8c--4531--b31b--e897642642b9 253:11   0     2G  0 lvm  /var/lib/kubelet/pods/b922fdce-33ed-40ab-b4b2-679f07ec97e5/volumes/kubernetes.io~csi/pvc-b7bc6400-4c8c-4531-b31b-e897642642b9/mount
├─LocalStorage_PoolHDD-pvc--77c8fe2f--a729--46cd--a030--3223eeb34440 253:12   0     1G  0 lvm
├─LocalStorage_PoolHDD-pvc--204aad0a--cf9b--4441--84dc--ef2cff1b0c73 253:13   0     1G  0 lvm  /mnt/hwameistor/dst/pvc-204aad0a-cf9b-4441-84dc-ef2cff1b0c73
└─LocalStorage_PoolHDD-pvc--61492e52--ee5a--4d3b--b3b0--90e6128b273d 253:14   0     2G  0 lvm  /var/lib/kubelet/pods/cd90182a-d9f4-4f1a-a460-8b3d67fa83c3/volumes/kubernetes.io~csi/pvc-61492e52-ee5a-4d3b-b3b0-90e6128b273d/mount
sr0                                                                   11:0    1  1024M  0 rom
```

Output of `blkid`:
```
[root@localhost ~]# blkid
/dev/sdb: UUID="3XEYZl-x1w1-f4O0-CTcQ-dueT-cgKM-kWNGd6" TYPE="LVM2_member"
/dev/sda1: UUID="893cdc71-7117-4dbf-abd2-d99eee9b2f31" TYPE="xfs"
/dev/sda2: UUID="eqJNHW-pDNe-6FT6-7Ddd-iU8P-Mfh3-GrgcRS" TYPE="LVM2_member"
/dev/mapper/centos-root: UUID="4199dc01-bf05-4c8e-997c-13b35fcd5ae0" TYPE="xfs"
/dev/mapper/centos-swap: UUID="b30a8140-8231-4cf7-a25e-8ded9a75f745" TYPE="swap"
/dev/mapper/centos-home: UUID="a4b8c0f6-0c3c-4c2b-90e6-3e2461136717" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--dae4f576--c03a--479c--ad9c--c107e83b3cc6: UUID="fe965f36-08f3-47b6-9dde-dfeee026f5e3" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--5835a84b--20c2--4cac--b677--90836732577d: UUID="ff076f1d-18b5-48ce-b134-a79999c225fd" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--fa14345c--0704--4646--9cb4--a91ee8ee54c9: UUID="99182516-2e7e-461b-9d85-7b01b368aecd" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--d49464f5--00a3--4638--9726--4c059baa12e2: UUID="826746a1-344d-43d2-9b9f-56701a4788b6" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--8627b40b--44a9--4e69--85a6--bec5c1d9b005: UUID="6405c6e6-ec8e-457e-ae9e-797ee8af7312" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--64ed9646--bc61--4d2c--91ff--a6108725f9a2: UUID="70d54b18-c925-4742-b8ae-211d9b33ba28" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--995ef500--6b84--4d65--81f9--425eb9795b7a: UUID="ccf2aab4-72bc-4e54-a365-bea8d1ccd384" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--b7bc6400--4c8c--4531--b31b--e897642642b9: UUID="6861c900-a74d-4295-a5fe-7a559c29d536" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--77c8fe2f--a729--46cd--a030--3223eeb34440: UUID="d1bfb4de-ddd7-4196-8d6e-b31ed38efd51" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--204aad0a--cf9b--4441--84dc--ef2cff1b0c73: UUID="d2719c93-702d-4189-a54b-62a187e6c8b1" TYPE="xfs"
/dev/mapper/LocalStorage_PoolHDD-pvc--61492e52--ee5a--4d3b--b3b0--90e6128b273d: UUID="6d6ededc-3df8-43fd-bb0c-31aadf4916e8" TYPE="xfs"
```

Output of `kubectl version`:
```
[root@localhost ~]# kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.0", GitCommit:"1b4df30b3cdfeaba6024e81e559a6cd09a089d65", GitTreeState:"clean", BuildDate:"2023-04-11T17:10:18Z", GoVersion:"go1.20.3", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v5.0.1
Server Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.0", GitCommit:"1b4df30b3cdfeaba6024e81e559a6cd09a089d65", GitTreeState:"clean", BuildDate:"2023-04-11T17:04:24Z", GoVersion:"go1.20.3", Compiler:"gc", Platform:"linux/amd64"}
```

Output of `kubectl get lsn -o yaml`:
```
[root@localhost ~]# kubectl get lsn -o yaml
apiVersion: v1
items:
- apiVersion: hwameistor.io/v1alpha1
  kind: LocalStorageNode
  metadata:
    creationTimestamp: "2023-08-29T02:42:06Z"
    generation: 1
    name: g-master1
    resourceVersion: "148379"
    uid: eca8d2a7-c4af-4f1d-9697-093f1fe22fa6
  spec:
    hostname: g-master1
    storageIP: 172.30.41.194
    topogoly:
      region: default
      zone: default
  status:
    conditions:
    - lastTransitionTime: "2023-08-29T02:44:56Z"
      lastUpdateTime: "2023-08-29T02:44:56Z"
      message: Successfully to expand storage capacity
      reason: StorageExpandSuccess
      status: "True"
      type: ExpandSuccess
    poolExtendRecords:
      LocalStorage_PoolHDD:
      - description:
          diskType: HDD
        diskRefs:
        - apiVersion: hwameistor.io/v1alpha1
          kind: LocalDisk
          name: g-master1-sdb
          resourceVersion: "4508"
          uid: 4be35b33-36da-41c3-b56f-e1dc0436e3fe
        nodeName: g-master1
        owner: local-storage
    pools:
      LocalStorage_PoolHDD:
        class: HDD
        disks:
        - capacityBytes: 113812439040
          devPath: /dev/sdb
          state: InUse
          type: HDD
        freeCapacityBytes: 8585740288
        freeVolumeCount: 989
        name: LocalStorage_PoolHDD
        totalCapacityBytes: 113812439040
        totalVolumeCount: 1000
        type: REGULAR
        usedCapacityBytes: 105226698752
        usedVolumeCount: 11
        volumeCapacityBytesLimit: 113812439040
        volumes:
        - pvc-8627b40b-44a9-4e69-85a6-bec5c1d9b005
        - pvc-d49464f5-00a3-4638-9726-4c059baa12e2
        - pvc-fa14345c-0704-4646-9cb4-a91ee8ee54c9
        - pvc-5835a84b-20c2-4cac-b677-90836732577d
        - pvc-64ed9646-bc61-4d2c-91ff-a6108725f9a2
        - pvc-dae4f576-c03a-479c-ad9c-c107e83b3cc6
        - pvc-995ef500-6b84-4d65-81f9-425eb9795b7a
        - pvc-b7bc6400-4c8c-4531-b31b-e897642642b9
        - pvc-77c8fe2f-a729-46cd-a030-3223eeb34440
        - pvc-204aad0a-cf9b-4441-84dc-ef2cff1b0c73
        - pvc-61492e52-ee5a-4d3b-b3b0-90e6128b273d
    state: Ready
- apiVersion: hwameistor.io/v1alpha1
  kind: LocalStorageNode
  metadata:
    creationTimestamp: "2023-08-29T02:42:06Z"
    generation: 1
    name: g-master2
    resourceVersion: "148343"
    uid: 654fe65e-8785-46da-b61c-e997993d9c6a
  spec:
    hostname: g-master2
    storageIP: 172.30.41.195
    topogoly:
      region: default
      zone: default
  status:
    conditions:
    - lastTransitionTime: "2023-08-29T02:44:56Z"
      lastUpdateTime: "2023-08-29T02:44:56Z"
      message: Successfully to expand storage capacity
      reason: StorageExpandSuccess
      status: "True"
      type: ExpandSuccess
    poolExtendRecords:
      LocalStorage_PoolHDD:
      - description:
          diskType: HDD
        diskRefs:
        - apiVersion: hwameistor.io/v1alpha1
          kind: LocalDisk
          name: g-master2-sdb
          resourceVersion: "4487"
          uid: 5214a959-60d1-4a27-8525-69669c5b9876
        nodeName: g-master2
        owner: local-storage
    pools:
      LocalStorage_PoolHDD:
        class: HDD
        disks:
        - capacityBytes: 113812439040
          devPath: /dev/sdb
          state: InUse
          type: HDD
        freeCapacityBytes: 3217031168
        freeVolumeCount: 990
        name: LocalStorage_PoolHDD
        totalCapacityBytes: 113812439040
        totalVolumeCount: 1000
        type: REGULAR
        usedCapacityBytes: 110595407872
        usedVolumeCount: 10
        volumeCapacityBytesLimit: 113812439040
        volumes:
        - pvc-27a7f00b-bc3d-4b14-a1f5-4185612f511f
        - pvc-2f075f6a-d586-4326-9f6a-aec169871b3a
        - pvc-77c8fe2f-a729-46cd-a030-3223eeb34440
        - pvc-2fb43f40-0622-4dc3-8438-834f4c845226
        - pvc-ea00cbd1-325a-42ab-b1aa-e91158bb992d
        - pvc-70c8953e-b6c5-4277-8fe7-e1754c2cc76f
        - pvc-5835a84b-20c2-4cac-b677-90836732577d
        - pvc-dae4f576-c03a-479c-ad9c-c107e83b3cc6
        - pvc-d49464f5-00a3-4638-9726-4c059baa12e2
        - pvc-64ed9646-bc61-4d2c-91ff-a6108725f9a2
    state: Ready
- apiVersion: hwameistor.io/v1alpha1
  kind: LocalStorageNode
  metadata:
    creationTimestamp: "2023-08-29T02:42:08Z"
    generation: 1
    name: g-master3
    resourceVersion: "130591"
    uid: 4e952262-6a65-4aae-8726-fab902a2d332
  spec:
    hostname: g-master3
    storageIP: 172.30.41.196
    topogoly:
      region: default
      zone: default
  status:
    conditions:
    - lastTransitionTime: "2023-08-29T02:44:55Z"
      lastUpdateTime: "2023-08-29T02:44:55Z"
      message: Successfully to expand storage capacity
      reason: StorageExpandSuccess
      status: "True"
      type: ExpandSuccess
    poolExtendRecords:
      LocalStorage_PoolHDD:
      - description:
          diskType: HDD
        diskRefs:
        - apiVersion: hwameistor.io/v1alpha1
          kind: LocalDisk
          name: g-master3-sdb
          resourceVersion: "4470"
          uid: de0c4730-3fbe-4c5b-8264-7d2f843b9604
        nodeName: g-master3
        owner: local-storage
    pools:
      LocalStorage_PoolHDD:
        class: HDD
        disks:
        - capacityBytes: 113812439040
          devPath: /dev/sdb
          state: InUse
          type: HDD
        freeCapacityBytes: 3217031168
        freeVolumeCount: 993
        name: LocalStorage_PoolHDD
        totalCapacityBytes: 113812439040
        totalVolumeCount: 1000
        type: REGULAR
        usedCapacityBytes: 110595407872
        usedVolumeCount: 7
        volumeCapacityBytesLimit: 113812439040
        volumes:
        - pvc-3529e1c3-d66d-4d5c-914f-dab6d6621204
        - pvc-6f1be3a6-7ff1-447c-98fb-bde830777a00
        - pvc-70a56d43-8f6f-41a1-96e4-683a8e73c45a
        - pvc-204aad0a-cf9b-4441-84dc-ef2cff1b0c73
        - pvc-2fb43f40-0622-4dc3-8438-834f4c845226
        - pvc-2f075f6a-d586-4326-9f6a-aec169871b3a
        - pvc-ea00cbd1-325a-42ab-b1aa-e91158bb992d
    state: Ready
kind: List
metadata:
  resourceVersion: ""
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
```
NAME        STATUS   ROLES           AGE     VERSION   INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                KERNEL-VERSION           CONTAINER-RUNTIME
g-master1   Ready    control-plane   3h18m   v1.27.0   194   <none>        CentOS Linux 7 (Core)   3.10.0-1160.el7.x86_64   containerd://1.7.2
g-master2   Ready    control-plane   3h17m   v1.27.0   195   <none>        CentOS Linux 7 (Core)   3.10.0-1160.el7.x86_64   containerd://1.7.2
g-master3   Ready    control-plane   3h16m   v1.27.0   196   <none>        CentOS Linux 7 (Core)   3.10.0-1160.el7.x86_64   containerd://1.7.2
```

**Comment by hangscer8** [comment]
error:

```
[root@localhost ~]# kubectl get pods -n hwameistor | grep -v Running
NAME                                                              READY   STATUS    RESTARTS      AGE
evictor-pvc-204aad0a-cf9b-datacopy-data-mysql-operator-2-6ngh9    0/1     Pending   0             80m
evictor-pvc-d49464f5-00a3-datacopy-data-mcamel-common-kpan6fj2l   0/1     Error     0             78m
evictor-pvc-d49464f5-00a3-datacopy-data-mcamel-common-kpanh5wsq   0/1     Error     0             81m
evictor-pvc-d49464f5-00a3-datacopy-data-mcamel-common-kpanxcv5d   0/1     Error     0             94m
```

```
[root@localhost ~]# kubectl logs evictor-pvc-d49464f5-00a3-datacopy-data-mcamel-common-kpan6fj2l -n hwameistor

2023-08-29 04:34:36 ERROR : mysql.sock: Failed to copy: failed to open source object: Open failed: file does not exist
Transferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                 1 (retrying may help)
Checks:                25 / 29, 86%
Transferred:            0 / 2, 0%
Elapsed time:         0.9s
Checking:
 *                             #ib_16384_0.dblwr: checking
 *                                   ib_logfile0: checking
 *                                      undo_001: checking
 *                                      undo_002: checking
Transferring:
 *                                    mysql.sock: transferring
 * mcamel-common-kpanda-m…uster-mysql-1-slow.log: transferringTransferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                 1 (retrying may help)
Checks:                27 / 31, 87%
Transferred:            0 / 1, 0%
Elapsed time:         1.4s
Checking:
 *                             #ib_16384_0.dblwr: checking
 *                                   ib_logfile0: checking
 *                                      undo_001: checking
 *                                      undo_002: checking
Transferring:
 * mcamel-common-kpanda-m…uster-mysql-1-slow.log: transferringTransferred:   	   20.687k / 20.687 kBytes, 100%, 20.844 kBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               138 / 142, 97%
Transferred:            1 / 1, 100%
Elapsed time:         1.9s
Checking:
 *                             #ib_16384_0.dblwr: checking
 *                                   ib_logfile0: checking
 *                                      undo_001: checking
 *                                      undo_002: checkingTransferred:   	   38.235M / 81.239 MBytes, 47%, 25.620 MBytes/s, ETA 1s
Errors:                 1 (retrying may help)
Checks:               169 / 169, 100%
Transferred:           13 / 14, 93%
Elapsed time:         2.4s
Transferring:
 *                                   ib_logfile0: 10% /48M, 0/s, -
2023-08-29 04:34:38 ERROR : sftp://root@g-master2:22//mnt/hwameistor/dst/pvc-d49464f5-00a3-4638-9726-4c059baa12e2: not deleting files as there were IO errors
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.045 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               169 / 169, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.8s
2023-08-29 04:34:38 ERROR : sftp://root@g-master2:22//mnt/hwameistor/dst/pvc-d49464f5-00a3-4638-9726-4c059baa12e2: not deleting directories as there were IO errors
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.045 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               169 / 169, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.8s
2023-08-29 04:34:38 ERROR : Attempt 1/3 failed with 1 errors and: failed to open source object: Open failed: file does not exist
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.045 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               169 / 169, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.8s
2023-08-29 04:34:38 ERROR : mysql.sock: Failed to copy: failed to open source object: Open failed: file does not exist
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.026 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               198 / 198, 100%
Transferred:           14 / 15, 93%
Elapsed time:         2.8s
Transferring:
 *                                    mysql.sock: transferring
2023-08-29 04:34:38 ERROR : sftp://root@g-master2:22//mnt/hwameistor/dst/pvc-d49464f5-00a3-4638-9726-4c059baa12e2: not deleting files as there were IO errors
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.003 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               338 / 338, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9s
2023-08-29 04:34:38 ERROR : sftp://root@g-master2:22//mnt/hwameistor/dst/pvc-d49464f5-00a3-4638-9726-4c059baa12e2: not deleting directories as there were IO errors
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.003 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               338 / 338, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9s
2023-08-29 04:34:38 ERROR : Attempt 2/3 failed with 1 errors and: failed to open source object: Open failed: file does not exist
Transferred:   	   81.239M / 81.239 MBytes, 100%, 42.003 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               338 / 338, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9s
2023-08-29 04:34:38 ERROR : mysql.sock: Failed to copy: failed to open source object: Open failed: file does not exist
Transferred:   	   81.239M / 81.239 MBytes, 100%, 41.989 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               367 / 367, 100%
Transferred:           14 / 15, 93%
Elapsed time:         2.9s
Transferring:
 *                                    mysql.sock: transferring
2023-08-29 04:34:38 ERROR : sftp://root@g-master2:22//mnt/hwameistor/dst/pvc-d49464f5-00a3-4638-9726-4c059baa12e2: not deleting files as there were IO errors
Transferred:   	   81.239M / 81.239 MBytes, 100%, 41.976 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               507 / 507, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9s
2023-08-29 04:34:38 ERROR : sftp://root@g-master2:22//mnt/hwameistor/dst/pvc-d49464f5-00a3-4638-9726-4c059baa12e2: not deleting directories as there were IO errors
Transferred:   	   81.239M / 81.239 MBytes, 100%, 41.976 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               507 / 507, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9s
2023-08-29 04:34:38 ERROR : Attempt 3/3 failed with 1 errors and: failed to open source object: Open failed: file does not exist
Transferred:   	   81.239M / 81.239 MBytes, 100%, 41.976 MBytes/s, ETA 0s
Errors:                 1 (retrying may help)
Checks:               507 / 507, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9sTransferred:   	   81.239M / 81.239 MBytes, 100%, 41.976 MBytes/s, ETA 0s
2023/08/29 04:34:38 Failed to sync: failed to open source object: Open failed: file does not exist
Errors:                 1 (retrying may help)
Checks:               507 / 507, 100%
Transferred:           14 / 14, 100%
Elapsed time:         2.9s
```

**Comment by sun7927** [comment]
can we access the test env for more info?

**Comment by sun7927** [comment]
checked the test env, the diff of src and dst volume is "lrwxrwxrwx. 1 polkitd input   27 Aug 30 04:21 mysql.sock -> /var/run/mysqld/mysqld.sock" which is a symbolic link. It looks like the rclone failed to copy this file.

**Comment by sun7927** [comment]
This issue should be solved already by replacing rclone with juicesync.

**Comment by sun7927** [comment]
can you test it with the HwameiStor latest version ?

**Comment by SSmallMonster** [comment]
@FloatXD does it work properly with HwameiStor latest version?

**Comment by SSmallMonster** [comment]
fixed in version `0.12.4`
